patches/minecraft/net/minecraft/inventory/container/EnchantmentContainer.java.patch

```diff
@@ -42,15 +42,15 @@
 -                           if (p_217002_2_.func_180495_p(p_217002_3_.func_177982_a(l * 2, 0, k)).func_203425_a(Blocks.field_150342_X)) {
 -                              ++j;
 -                           }
--
+ 
 -                           if (p_217002_2_.func_180495_p(p_217002_3_.func_177982_a(l * 2, 1, k)).func_203425_a(Blocks.field_150342_X)) {
 -                              ++j;
 -                           }
--
+ 
 -                           if (p_217002_2_.func_180495_p(p_217002_3_.func_177982_a(l, 0, k * 2)).func_203425_a(Blocks.field_150342_X)) {
 -                              ++j;
 -                           }
--
+ 
```

Comment:
Why?

Reply:
Removing uneccesary whitespace changes? Oh it does look like I missed one.